examples/webgl_multiple_views_multiview.html

```diff
@@ -0,0 +1,294 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js performance - animation</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				font-family: Monospace;
+				background-color: #000;
+				margin: 0px;
+				overflow: hidden;
+			}
+			#info {
+				position: absolute;
+				top: 10px;
+				width: 100%;
+				text-align: center;
+				z-index: 100;
+				display:block;
+			}
+			#info p {
+				max-width: 600px;
+				margin-left: auto;
+				margin-right: auto;
+				padding: 0 2em;
+			}
+			#info a {
+				color: #2fa1d6;
+				font-weight: bold;
+			}
+			.dg.ac {
+				z-index: 999 !important;
+			}
+		</style>
+	</head>
+
+	<body>
+
+		<div id="container"></div>
+		<div id="info">
+			<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - multiple views using multiview extension - by <a href="https://twitter.com/fernandojsg">fernandojsg</a><br/>
+			<span id="options"></span>
+		</div>
+
+		<script src="../build/three.js"></script>
+		<script src="js/loaders/GLTFLoader.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+		<script>
+
+			var container, clock, stats;
+			var camera, scene, renderer, panel, balls;
+			var radius = 0.08;
+
+			var mixers = [];
+			var cameras = [];
+
+			init();
+
+			function addNumLink( value ) {
+
+				return `<a href="?num=${ value }">${ value }</a>`;
+
+			}
+
+			function init() {
+
+				const urlParams = new URLSearchParams(window.location.search);
+
+				container = document.getElementById('container');
+
+				var options = document.getElementById('options');
+				options.innerHTML =`Number of balls:
+						${addNumLink(200)}
+						${addNumLink(500)}
+						${addNumLink(1000)}
+						${addNumLink(2500)}
+						${addNumLink(5000)}`;
+
+
+				var ASPECT_RATIO = window.innerWidth / window.innerHeight;
+				var AMOUNT = 2;
+				var SIZE = 1 / AMOUNT;
+				var WIDTH = ( window.innerWidth / AMOUNT ) * window.devicePixelRatio;
+				var HEIGHT = ( window.innerHeight / AMOUNT ) * window.devicePixelRatio;
+
+				for ( var y = 0; y < AMOUNT; y ++ ) {
+
+					for ( var x = 0; x < AMOUNT; x ++ ) {
+
+						var subcamera = new THREE.PerspectiveCamera( 40 + (x + y) * 20, ASPECT_RATIO, 0.25, 100 );
+						subcamera.bounds = new THREE.Vector4( x / AMOUNT, y / AMOUNT, SIZE, SIZE );
```

Comment:
https://github.com/mrdoob/three.js/issues/16361#issuecomment-488054088

Reply:
How about merging this PR as is so far (if looking good) and then considering to switch from `ArrayCamera` bounds to `ArrayCamera` viewport (if needed) in another PR? That'd be easier to discuss and review maybe.

Reply:
Addressed on https://github.com/mrdoob/three.js/pull/16367 once that will get merged I'll push the changed to fix this PR

Reply:
Updated it https://github.com/mrdoob/three.js/pull/16316/commits/3c58278708626269ef64c2db549e12ad4ca2a606#diff-c0e88b98497597a015ecf238e91ac3a0 /cc @mrdoob @takahirox